src/Sulu/Bundle/CoreBundle/Resources/config/doctrine.xml

```diff
@@ -5,6 +5,8 @@
     <services>
         <service id="sulu_core.doctrine.references" class="Sulu\Component\Doctrine\ReferencesOption">
             <argument type="service" id="doctrine"/>
+            <!-- the $targetEntityMapping argument is set by the ResolveTargetEntitiesPass -->
+            <argument key="$targetEntityMapping" type="collection" />
```

Comment:
i did not find any documentation about the `key` argument - not sure if it works with older symfony versions